localization/l10n/bundle.l10n.json

```diff
@@ -256,6 +256,8 @@
   "Save as CSV": "Save as CSV",
   "Save as Excel": "Save as Excel",
   "Save as JSON": "Save as JSON",
+  "No result found for the current active editor, please run a query or switch to another editor.": "No result found for the current active editor, please run a query or switch to another editor.",
```

Comment:
How about:

> No result found for the active editor; please run a query or switch to another editor.

removes the redundancy between "current" and "active" and switches the comma to a semicolon since they're different clauses.

Reply:
updated